content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -20,11 +20,15 @@ This page provides an overview of authenticating.
 All Kubernetes clusters have two categories of users: service accounts managed
 by Kubernetes, and normal users.
 
-Normal users are assumed to be managed by an outside, independent service. An
-admin distributing private keys, a user store like Keystone or Google Accounts,
-even a file with a list of usernames and passwords. In this regard, _Kubernetes
-does not have objects which represent normal user accounts._ Normal users
-cannot be added to a cluster through an API call.
+Normal users are assumed to be managed by
+
+- an outside, independent service
+- an administrator distributing private keys
+- a user store like Keystone or Google Accounts
+- a file with a list of usernames and passwords
+
+In this regard, _Kubernetes does not have objects which represent normal user
```

Comment:
Hi. I read through the updates. The first statement may be true, but the sentence that follows is confusing.
The second sentence compares a service to an "admin, key-value store, file system".
This change makes the distinction that a separate service could be used to manage users in addition to other non-service methods such as a human administrator managing users/passwords, files, key-value stores.
I think the original author meant to say:
It is assumed that a cluster-independent service manages normal users in the following ways:
- an administrator to distribute private keys
- a user store like ...
- a file system with a list ...

Reply:
Okay, revised as per suggested.

Reply:
@sftim, @jimangel, How does this sound to you?

Reply:
The text still seems a bit odd because it is comparing an "adminstrator distributing keys" to user account storage systems.

It is assumed that a cluster-independent service manages normal users. This service may be
used by an administrator to distribute private keys or the service may store user
account information in a user store like Keystone, Google Accounts, or in a file system as a list of
usernames and passwords.

Does this help?